link_telemetry.py

```diff
@@ -296,28 +271,26 @@ def process_response(future: concurrent.futures.Future):
     print(f"{ANSI_BOLD}Response HTTP status code:{ANSI_ESCAPE} {ANSI_GREEN}{response.status_code}{ANSI_ESCAPE}")
     
     parse_response: dict = response.json()
-       
     if parse_response["result"] == "OK":
         table = PrettyTable()
-        table.field_names = ["ID", "Source", "Class", "Measurement", "Value"]
-        measurements: list = parse_response["measurements"]
 
-        # format response as a table
-        for measurement in measurements:
-            id = parse_response['id']
-            table.add_row([hex(id), measurement["source"], measurement["m_class"], measurement["name"], measurement["value"]])
+        extracted_measurements = parse_response["message"]
```

Comment:
We'd want to have the "type" as a column in the pretty table - currently, it looks like the table is just based off the values associated with the "display_data" key in the data member of the newly-created message.